pos_session_pay_invoice/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+#.  Go to *Point of Sale / Dashboard* and create and open or access to an
+    already open POS Session.
+#.  Open the POS Session form view on the Backend.
+#.  Press the button **Pay Invoice** to pay a Supplier Invoice or a Customer
+    Refund. It will be paid using Cash.
```

Comment:
```suggestion
#.  Press the button **Pay Supplier Invoice** to pay a Supplier Invoice or **Pay Refund** for a Customer
    Refund. It will be paid using Cash.
```